domino_data/auth.py

```diff
@@ -44,6 +73,15 @@ def get_headers(self) -> Dict[str, str]:
         if self.api_key:
             self.headers["X-Domino-Api-Key"] = self.api_key
 
+        if self.token_url is not None:
+            try:
+                jwt = get_jwt_token(self.token_url)
+            except httpx.HTTPStatusError:
+                pass
```

Comment:
what’s the rationale behind passing in the case of an http error?

Reply:
try with the token file (L85) if there is an error, early return if not (L83)